packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -37,9 +37,9 @@ import {
   type WalletFactoryTestContext,
 } from './walletFactory.js';
 
-const DENOM_UNIT = 1n * 1_000_000n; // 1 million
+const DENOM_UNIT = 1n * 1_000_000n; // 6 decimal places
 
-const LIQUIDITY_POOL_SIZE = 500n * DENOM_UNIT; // 500 USDC
+const LIQUIDITY_POOL_SIZE = 1_000n * DENOM_UNIT; // $1,000 USDC
```

Comment:
the units should be either `$` or `USDC`, but not both, yes?

Reply:
I don't see the harm in using both but if you prefer one I'd drop the USDC since `$` is more concise and is equivalent in amount

Reply:
I don't feel strongly about it.